<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PresetsManager</class>
 <widget class="QDialog" name="PresetsManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Presets Manager</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="0" colspan="3">
    <widget class="QLabel" name="labelMessage">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Apply preset:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Preset:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>87</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButtonExportPreset">
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Export</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="export"/>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="pushButtonImportPreset">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Import</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="import"/>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="pushButtonDeletePreset">
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Delete</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="delete"/>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QComboBox" name="comboBoxPreset">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="6">
       <widget class="QListWidget" name="listWidgetTextures"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="pushButtonAplyPreset">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Save current preset:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEditPresetName"/>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QTreeWidget" name="treeWidget">
        <column>
         <property name="text">
          <string>Controls</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Code</string>
         </property>
        </column>
        <item>
         <property name="text">
          <string>Normal</string>
         </property>
         <property name="checkState">
          <enum>Checked</enum>
         </property>
         <item>
          <property name="text">
           <string>Enhance</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <item>
           <property name="text">
            <string>Height</string>
           </property>
           <property name="checkState">
            <enum>Checked</enum>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Soft</string>
           </property>
           <property name="checkState">
            <enum>Checked</enum>
           </property>
           <property name="text">
            <string>1</string>
           </property>
          </item>
         </item>
         <item>
          <property name="text">
           <string>Bump</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <item>
           <property name="text">
            <string>Height</string>
           </property>
           <property name="checkState">
            <enum>Checked</enum>
           </property>
           <property name="text">
            <string>2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Distance</string>
           </property>
           <property name="checkState">
            <enum>Checked</enum>
           </property>
           <property name="text">
            <string>3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Soft</string>
           </property>
           <property name="checkState">
            <enum>Checked</enum>
           </property>
           <property name="text">
            <string>4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Cut</string>
           </property>
           <property name="checkState">
            <enum>Checked</enum>
           </property>
           <property name="text">
            <string>5</string>
           </property>
          </item>
         </item>
         <item>
          <property name="text">
           <string>Invert</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <item>
           <property name="text">
            <string>X Axis</string>
           </property>
           <property name="checkState">
            <enum>Checked</enum>
           </property>
           <property name="text">
            <string>7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Y Axis</string>
           </property>
           <property name="checkState">
            <enum>Checked</enum>
           </property>
           <property name="text">
            <string>8</string>
           </property>
          </item>
         </item>
         <item>
          <property name="text">
           <string>Use Alpha</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
          <property name="text">
           <string>30</string>
          </property>
         </item>
        </item>
        <item>
         <property name="text">
          <string>Tile</string>
         </property>
         <property name="checkState">
          <enum>Checked</enum>
         </property>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Parallax</string>
         </property>
         <property name="checkState">
          <enum>Checked</enum>
         </property>
         <item>
          <property name="text">
           <string>Type</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
          <property name="text">
           <string>9</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Binary</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <item>
           <property name="text">
            <string>Threshold</string>
           </property>
           <property name="checkState">
            <enum>Checked</enum>
           </property>
           <property name="text">
            <string>10</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Focus</string>
           </property>
           <property name="checkState">
            <enum>Checked</enum>
           </property>
           <property name="text">
            <string>11</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Minimum Height</string>
           </property>
           <property name="checkState">
            <enum>Checked</enum>
           </property>
           <property name="text">
            <string>13</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Erode / Dilate</string>
           </property>
           <property name="checkState">
            <enum>Checked</enum>
           </property>
           <property name="text">
            <string>14</string>
           </property>
          </item>
         </item>
         <item>
          <property name="text">
           <string>Soft</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
          <property name="text">
           <string>12</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Height Map</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
          <item>
           <property name="text">
            <string>Brightness</string>
           </property>
           <property name="checkState">
            <enum>Checked</enum>
           </property>
           <property name="text">
            <string>15</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Contrast</string>
           </property>
           <property name="checkState">
            <enum>Checked</enum>
           </property>
           <property name="text">
            <string>16</string>
           </property>
          </item>
         </item>
         <item>
          <property name="text">
           <string>Invert</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
          <property name="text">
           <string>17</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Use Alpha</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
          <property name="text">
           <string>33</string>
          </property>
         </item>
        </item>
        <item>
         <property name="text">
          <string>Specular</string>
         </property>
         <property name="checkState">
          <enum>Checked</enum>
         </property>
         <item>
          <property name="text">
           <string>Soft</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
          <property name="text">
           <string>18</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Brightness</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
          <property name="text">
           <string>19</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Contrast</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
          <property name="text">
           <string>20</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Threshold</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
          <property name="text">
           <string>21</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Invert</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
          <property name="text">
           <string>22</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Use Alpha</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
          <property name="text">
           <string>31</string>
          </property>
         </item>
        </item>
        <item>
         <property name="text">
          <string>Occlussion</string>
         </property>
         <property name="checkState">
          <enum>Checked</enum>
         </property>
         <item>
          <property name="text">
           <string>Soft</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
          <property name="text">
           <string>23</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Brightness</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
          <property name="text">
           <string>24</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Invert</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
          <property name="text">
           <string>25</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Threshold</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
          <property name="text">
           <string>26</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Contrast</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
          <property name="text">
           <string>27</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Distance</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
          <property name="text">
           <string>28</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Distance Mode</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
          <property name="text">
           <string>29</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Use Alpha</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
          <property name="text">
           <string>32</string>
          </property>
         </item>
        </item>
        <item>
         <property name="text">
          <string>Lights</string>
         </property>
         <property name="checkState">
          <enum>Unchecked</enum>
         </property>
         <property name="text">
          <string>34</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="pushButtonSavePreset">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
